idaes/gas_solid_contactors/properties/methane_iron_OC_reduction/gas_phase_thermo.py

```diff
@@ -789,3 +933,21 @@ def default_material_balance_type(blk):
 
     def default_energy_balance_type(blk):
         return EnergyBalanceType.enthalpyTotal
+
+    def calculate_scaling_factors(self):
+        super().calculate_scaling_factors()
+
+        if self.is_property_constructed("sum_component_eqn"):
+            for t, v in self.sum_component_eqn.items():
+                iscale.constraint_scaling_transform(
+                    v,
+                    iscale.get_scaling_factor(self.mole_frac_comp[t]),
+                    overwrite=False)
```

Comment:
This is only one equation (`1 = sum(x_i)`), so the `for loop` isn't required. The component index should also be specified i.e. choose one of the components from the list.